css/main.css

```diff
@@ -1,3 +1,472 @@
+    align-items: center;
+    column-gap: 10px;
+    font-weight: 900;
+        grid-area: h1;
+        grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 1;
+    grid-row-end: 2;
+}
+.header-headline > h1, .front h2{
+    font-family:var(--font1);
+    font-weight: bold;
+    font-size: 2rem;
+}
+
+
+    .forward-slash{
+        font-size: 30px;
+        color: var(--icon-color);
+        font-weight: bold;
+        display: inline;
+    }
+    
+    .header-headline p, .front p{
+    font-family:var(--font_3);
+}
+    header p, .front p{
+        font-weight: lighter;
+        font-size: 1.7rem;
+    }
+    article>h2{
+        font-size: 10rem;
+    }
+    article h3{
+    font-size: 2.5rem;
+    font-weight: 500;
+
+    }
+    article p{
+        font-family: var(--font2);
+        line-height: 35px;
+        font-size: 1.5rem;
+    
+    }
+nav{
+    display: inline;
+    grid-area: nav;
+    grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        display: flex;  
+        justify-content: flex-end;
+        padding-right: 2%;
+}
+nav ul {
+        display: flex;
+        align-items: center;
+        column-gap: 20px;
+}
+nav ul li {
+    list-style: none; 
+    font-size: 1.5rem;
+    font-weight: 400;
+}
+
+nav ul li:hover {
+    color: blue;
+  }
+main{
+    grid-area: main;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 100vh;
+    display: grid;
+    grid-template-areas:
+            "first art";
+        grid-template-rows: 1fr;
+        grid-template-columns: 1fr 1fr;
+}
+
+.first-section{
+    grid-area: first;
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    position: relative;
+      height: 100vh;
+    }
+    .back{
+    position: absolute;
+    background: linear-gradient(180deg, #E6DACD 100%, white 50%);
+    height: 100vh;
+    }
+    .front{
+    position: absolute;
+      left: 10%;
+      background-color: var(--card-color);
+      width: 100%;
+      min-height: 60vh;
+      top: 20%;
+      bottom: 0%;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      box-shadow: -10px 5px 50px -15px grey;
+    }
+    .front p{
+        display: inline;
+    }
+    .front button{
+display: none;
+    }
+    img{
+        height: 250px;
+        object-fit: cover;
+    }
+    .icon{
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        gap: 20px;
+        background-color: var(--bg-color);
+        font-size: 2rem;
+    }
+    article{
+        grid-area: art;
+        grid-column-start: 2;
+        grid-column-end: 3;
+        grid-row-start: 1;
+        grid-row-end: 2;
+        background-color: var(--bg-color);
+        margin: 35% 0 0 5%; 
+        min-height: 60vh;
+        display: flex;
+        flex-direction: column;
+        row-gap: 3%;
+    }
+    .front p{
```

Comment:
at the moment your margins are quite large which is pushing your circular image up out of the card, if you set the height as `auto` like mentioned above and reduce these margins this should help